### PR TITLE
Javascript javascript week2

### PR DESCRIPTION
**Pull request by AnnaDzig**

### CODE REVIEW

File: javascript/javascript1/week2/script.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        return `Mis ${firstname} ${surname}`
+    }else {
+        return `${firstname} ${surname}`
+    }
+}
+
+let fullname1 = gettFullname('Leonardo', 'DiCaprio', true, 'male')
+let fullname2 = gettFullname('Angellina', 'Jolie', true, 'female')
+let fullname = gettFullname('Joey', 'Ttibbiani', false, 'male')
+
```
Comment (vulk29) [inline javascript/javascript1/week2/script.js:16]: nice! :) you can also console.log your function so you can see your returns. 
So you can simply add a console.log(fullname1)

File: javascript/javascript1/week2/script.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    } else {
+        advice = 'It is hot  outside, take off your clothes and go to the beach '
+    }
+
+    return advice;
+}
+
+temperatureOutside(18)
+
+//Student manager
```
Comment (vulk29) [inline javascript/javascript1/week2/script.js:62]: nice!! 👍

File: javascript/javascript1/week2/script.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            } else {
+                return alert ("Cannot add more students to class 07");
+            }
+        }
+    } else {
+        return alert ("Please enter student name");
+    }
+
+}
+
```
Comment (vulk29) [inline javascript/javascript1/week2/script.js:93]: great use of nested ifs!

File: javascript/javascript1/week2/script.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        let futureEventIndexLover = day + event;
+        return `The event will be on ${dayTitle[futureEventIndexLover]}`;
+    }else {
+        console.log('incorrect')
+    }
+}
+}
+getEventWeekday(12)
+
+
```
Comment (vulk29) [inline javascript/javascript1/week2/script.js:40]: i like this check for number or not, but it doesn't quite work as intended. also, if the event is 7 days from now your function returns 'incorrect' ;)

File: javascript/javascript1/week2/script.js [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  } else {
+      console.log("Enough candy for you!");
+  }
+
+}
+console.log(addCandy("mint", 10));
+console.log(addCandy("toffee", 50));
+console.log(addCandy("chewing-gum", 10));
+console.log(addCandy("chocolate", 40));
+canBuyMoreCandy();
```
Comment (vulk29) [inline javascript/javascript1/week2/script.js:132]: you should really add console.logs to all your scripts to test all corner cases, as you have done here!

**Review by vulk29 — CHANGES_REQUESTED** [review]
great job overall, it is only your getEventWeekday that needs a bit of attention. also, add more console.logs to test your scripts :)
